### DISCUSSION TITLE
permit to set a border on picture

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#6032] [Ideas]
I have pictures with majority of white on the border and its hard to distinguish them, moreover when 2 pictures are following.
To improve legibility that would be great to have small border around.

**To Reproduce**

**Expected behavior**

**Screenshots**
![before](https://github.com/outline/outline/assets/80455946/3ed68e80-c3cb-460f-8ecf-9eb669ced4dc)
current

![after](https://github.com/outline/outline/assets/80455946/0e1f5542-c934-448f-a2e0-0285e0abe8a7)
expected

**Outline (please complete the following information):**
- Install: self hosted
- Version: 0.72.1

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: firefox
 - Version: 118

**Comment by Limezy** [comment]
I have seen some discussions here where outline users were changing a little bit the css to adapt the look of it to their taste. Maybe you could look at that ?
